src/docs/goose-book/src/config/defaults.md

```diff
@@ -81,8 +81,8 @@ For example, without any run-time options the following load test would automati
             .register_transaction(transaction!(loadtest_index))
         )
         .set_default(GooseDefault::Host, "local.dev")?
-        .set_default(GooseDefault::RequestsFile, "goose-requests.log")?
-        .set_default(GooseDefault::DebugFile, "goose-debug.log")?
+        .set_default(GooseDefault::RequestsLog, "goose-requests.log")?
```

Comment:
This is `RequestLog` not `RequestsLog` (drop the `s`):
https://docs.rs/goose/*/goose/config/enum.GooseDefault.html#variant.RequestLog